src/Streamly/Internal/Data/Time/TimeSpec.hsc

```diff
@@ -24,7 +26,7 @@ module Streamly.Internal.Data.Time.TimeSpec
 where
 
 import Data.Int (Int64)
-#if i386_HOST_ARCH
+#if (WORD_SIZE_IN_BITS == 32)
```

Comment:
Is this conditional necessary ? Given that staging-v0.7.3 branch built successfully on arm-v7 (and it doesn't have this flag) I am thinking this won't be required

Reply:
Will result in unused import warning, don't remember seeing it in staging-v0.7.3. branch build. I'll merge this.